inc/classes/class-srm-post-type.php

```diff
@@ -570,6 +577,7 @@ public function redirect_rule_metabox( $post ) {
 			<label for="srm_redirect_rule_to"><strong><?php esc_html_e( '* Redirect To:', 'safe-redirect-manager' ); ?></strong></label><br />
 			<input class="widefat" type="text" name="srm_redirect_rule_to" id="srm_redirect_rule_to" value="<?php echo esc_attr( $redirect_to ); ?>" />
 		</p>
+		<p class="description" id="srm_to_disabled_message" style="display:none;"><em><?php esc_html_e( 'No "Redirect to" value applies for 4xx error codes.', 'safe-redirect-manager' ); ?></em></p>
```

Comment:
I'd suggest tweaking this text a bit:

```suggestion
		<p class="description" id="srm_to_disabled_message" style="display:none;"><em><?php esc_html_e( 'The "Redirect to" value doesn\'t apply for 4xx error codes.', 'safe-redirect-manager' ); ?></em></p>
```